### DISCUSSION TITLE
自定义条件表达式

### DISCUSSION THREAD

**Discussion by didoodidoo** [bfenetworks/bfe#1001] [Ideas（新功能的建议和讨论）]
当前不支持以插件的形式添加自定义的条件表达式吧?后续会考虑添加这方面的扩展吗?

**Comment by iyangsj** [comment]
这类需求非常特殊，目前在文档中并未说明。

代码实现中包括统一的 [req_context_value_in() ](https://github.com/bfenetworks/bfe/blob/develop/bfe_basic/condition/build.go#L520)条件原语来间接支持这类需求（避免条件原语数量过多），即：
- 在请求处理过程中，扩展插件按需在请求上下文中添加标记<tagName，tagValue>
- 在请求后续处理时，基于req_context_value_in() 原语进行判定

后续版本文档将增加示例进行说明。

**Reply by didoodidoo** [reply]
好的,感谢回复.
